onnx/defs/controlflow/defs.cc

```diff
@@ -12,18 +12,18 @@
 namespace ONNX_NAMESPACE {
 using SupportType = OpSchema::SupportType;
 
-static std::vector<std::string> control_flow_types_ir9() {
-  auto t = OpSchema::all_tensor_types_ir9();
```

Comment:
I think we should keep them to avoid the existing code taking a dependency on the c header of onnx to break. onnxruntime would break.